### ISSUE TITLE
Nothing happens

### ISSUE THREAD

**Issue by dentroboy** [mathewleland/export-svg-with-styles#1]
Hey,

I implemented code like in the example:

```
handleClick = () => {
		const element = document.getElementById("test");
		console.log('test', element);
		const options = {
			width: 300,
			height: 300,
			svg: element,
			filename: 'test.png'
		};

		downloadSVG(options);
	}
```
And I have some react-vis charts in my div:

```
<div id="test" style={{ padding: 3, paddingLeft: 10 }}>
						<XYPlot
							width={800}
							height={300}
							dontCheckIfEmpty
							yDomain={[minTempRounded, maxTempRounded]}
							onMouseLeave={() => this.setState({ crosshairTemp: [] })}
						>
							<HorizontalGridLines />
							<VerticalGridLines />
							<YAxis />
...
</div>
```
Then I have a simple button which calls handleClick, but nothing happens when I click.
What am I doing wrong?

**Comment by mathewleland** [comment]
Thanks for reaching out, and this is a good question.  I use this library with React-vis as well! And the function has to take an SVG element, not a div element.  So in this case, you could use `const element = document.querySelector("#test svg")` and that should resolve it for you.  If you only have one SVG on the page, it defaults to just grabbing the first SVG available, so you could leave the svg option out if you even wanted to.

Let me know if this solves your problem!

**Comment by dentroboy** [comment]
Hey mathew, thanks for the response! Yeah I changed my code like you mentioned and it worked. But the image doesnt look like I wanna have it so I decided to use another libary to handle saving the graph as an image. I use [vega](https://vega.github.io/vega/) now and generate an svg via a json, server-side. Thats easier for me because I need it later on to generate a pdf with the graph in it.

But definitely thanks and props to you!
